pkg/plugin/v3/scaffolds/internal/templates/config/kdefault/config_manager_patch.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Could you rename this to `manager_config_patch.go`? The struct is `ManagerConfigPatch` and the scaffolded file `manager_config_patch.yaml`.

Reply:
Absolutely, I was noticing that the directory where this lives had some mixed struct to filename associations. see https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/v3/scaffolds/internal/templates/config/kdefault/webhook_manager_patch.go and https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/v3/scaffolds/internal/templates/config/kdefault/enablecainection_patch.go

Potentially in another PR we should clean those up.

Reply:
I did a first round cleaning up things but there are some left yes